site/app/views/course/CourseMaterialsView.php

```diff
@@ -77,7 +88,7 @@ public function listCourseMaterials(array $course_materials_db) {
             if ($course_material->isDir()) {
                 continue;
             }
-            if ($this->core->getUser()->getGroup() != 1 && $course_material->getReleaseDate() > $date_now) {
+            if ($this->core->getUser()->getGroup() > 3 && $course_material->getReleaseDate() > $date_now) {
```

Comment:
I think using `if( ! $this->core->getUser()->accessFullGrading() )` is a more readable check

https://github.com/Submitty/Submitty/blob/master/site/app/models/User.php#L349-L351